install_config/storage_examples/azure_blob_docker_registry_example.adoc

```diff
@@ -0,0 +1,101 @@
+[[azure-blob-docker-registry]]
+= Configuring Azure Blob Storage for Integrated Docker Registry
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+toc::[]
+
+[[azure-blob-docker-registry-overview]]
+== Overview
+This topic reviews how to configure Azure Blob Storage for OpenShift integrated Docker Registry.
```

Comment:
Microsoft Azure Blob storage for the OpenShift-integrated Docker Registry (or do you mean [OpenShift Container Registry](https://doc-stage.usersys.redhat.com/documentation/en-us/ccs_internal_documentation/1.0/html-single/glossary_of_terms_and_conventions_for_product_documentation/#openshift-container-registry)?)